packages/mdc-dialog/tests/dummy/app/templates/application.hbs

```diff
@@ -16,7 +16,7 @@
 {{#mdc-dialog title="Choose a Ringtone"
               scrollable=true
               show=showScrollableDialog
-              positiveButton=(hash action="accept" label="Accept" closed=(action "accept"))
+              positiveButton=(hash action="accept" label="Accept" closed=(action "accept") disabled=false)
```

Comment:
@jamesstaub Is is necessary to explicitly put `disabled=false`? The way Ember works, `disabled` will default to `false`. Or, are you just illustrating this in the example for demonstration purposes?

Reply:
oh it was just to demonstrate that it's available. happy to remove that if you'd prefer.

Reply:
Lets only explicitly set one of the examples to false so developers do ni think it is required.